operators/kiali/1.41.0/manifests/kiali.v1.41.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,541 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
It will not work on 4.9

Reply:
Is this new? First I heard of our need to bump up the CSV apiVersion for 4.9.